docs/reference/deployment_options.md

```diff
@@ -32,6 +32,35 @@ Terraform module to deploy the 5G user plane in edge sites.
 
 `````
 
+## User Plane Function
+
+The User Plane Function (UPF) is available in two different charms:
+
+`````{tab-set}
+    
+````{tab-item} Kubernetes
+
+The [UPF Kubernetes Charm](https://charmhub.io/sdcore-upf-k8s) can be deployed on a Kubernetes cluster. Network Configuration is done using Multus. Unless the charm is deployed in DPDK mode, the charm will not modify the host network configuration. To deploy the Kubernetes charm, follow this [guide](/how-to/deploy_sdcore_cups/).
```

Comment:
The part about modifying the network configuration is more or less correct. Using macvlan or DPDK requires changes to the host for preparation. DPDK is more involved, but with macvlan we need to manually create the macvlan interfaces on the host. Those changes are not done by the charm however, because it can't.

Reply:
I tried to improve it, let me know what you think